package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "smoke-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "smoke.js",
+  "dependencies": {
```

Comment:
we should probably add `"private": true` to prevent accidental publication: https://docs.npmjs.com/files/package.json#private